src/credentials/FabricTable.h

```diff
@@ -1003,6 +1003,13 @@ class DLL_EXPORT FabricTable
      */
     CHIP_ERROR PeekFabricIndexForNextAddition(FabricIndex & outIndex);
 
+    /**
+     * Set the fabric index that will be used fo the next fabric added.
```

Comment:
This should document that use of this API might well be a spec violation (spec requires sequential assignment) and hence it must be used with extreme care.

Reply:
If there's a spec violation but nobody can prove it, does it matter? Removing fabrics is permitted.

Reply:
Added a clarifying comment that this may affect conformance.